jsk_perception/CMakeLists.txt

```diff
@@ -228,6 +229,7 @@ jsk_add_nodelet(src/project_image_point.cpp "jsk_perception/ProjectImagePoint" "
 jsk_add_nodelet(src/sliding_window_object_detector.cpp "jsk_perception/SlidingWindowObjectDetector" "sliding_window_object_detector")
 jsk_add_nodelet(src/gaussian_blur.cpp "jsk_perception/GaussianBlur" "gaussian_blur")
 jsk_add_nodelet(src/kmeans.cpp "jsk_perception/KMeans" "kmeans")
+jsk_add_nodelet(src/darknet_object_segmentation "jsk_perception/DarknetObjectSegmentation" "darknet_object_segmentation")
```

Comment:
I think `ObjectDetection` is more standard name in CV field, and `ObjectSegmentation` is recognition in pixel-wise.
![image](https://cloud.githubusercontent.com/assets/4310419/20838664/e2a0fdc8-b8ec-11e6-9dba-01483fa45b22.png)